nemo/collections/multimodal/speech_llm/data/lhotse_dataset.py

```diff
@@ -140,7 +140,7 @@ def collate_text_data(
             k: torch.as_tensor(v)
             for k, v in text_processor._process_example(
                 context=cut.context,
-                output=cut.supervisions[0].text,
+                output=str(cut.supervisions[0].text),
```

Comment:
It should have been `cut.supervisions[0].text if cut.supervisions[0].text is not None else ""` otherwise you will get string `"None"` for supervisions without text...